mods/voidmulti.sc2mod/base.sc2data/gamedata/upgradedata.xml

```diff
@@ -46,7 +46,7 @@
         <Icon value="Assets\Textures\btn-upgrade-terran-hyperflightrotors.dds"/>
         <Alert value="ResearchComplete"/>
         <Race value="Terr"/>
-        <ScoreAmount value="200"/>
+        <ScoreAmount value="250"/>
```

Comment:
Hyperflight Rotors cost reduced from 150/150 to 125/125

This changes the score value that you receive for researching the upgrade. The score is visible in the stats after the match. The score value should always match the resource costs.
The value before was much too low. Now it is correct for the new costs